### PR TITLE
Fix OCS unit tests after two share merges

### PR DESCRIPTION
**Pull request by rullzer**
Merging of https://github.com/owncloud/core/pull/21957 and https://github.com/owncloud/core/pull/21960 created some conflicts.

Should all be fixed now!

CC: @PVince81 @MorrisJobke @DeepDiver1975 @nickvergessen @schiesbn @LukasReschke

### CODE REVIEW

File: apps/files_sharing/tests/api/share20ocstest.php [hunk 1]
```diff
@@ -806,15 +806,15 @@ public function testCreateShareLinkPublicUploadFolder() {
 		$currentUser = $this->currentUser;
 
 		$this->shareManager->expects($this->once())->method('createShare')->with(
-			$this->callback(function (IShare $share) use ($path, $currentUser) {
-				return $share->getPath() === $path &&
+			$this->callback(function (\OCP\Share\IShare $share) use ($path, $currentUser) {
+				return $share->getNode() === $path &&
```
Comment (icewind1991) [inline apps/files_sharing/tests/api/share20ocstest.php]: `Node === string` ?

Comment (nickvergessen) [inline apps/files_sharing/tests/api/share20ocstest.php]: `$path = $this->getMock('\OCP\Files\Folder');`

**Comment by icewind1991** [comment]
Fixes the tests :+1:

**Comment by PVince81** [comment]
:+1:

**Pull request merged**